src/corehost/cli/hostpolicy/hostpolicy.cpp

```diff
@@ -560,7 +561,7 @@ namespace
 
     bool matches_existing_properties(const corehost_initialize_request_t *init_request)
     {
-        const hostpolicy_context_t *context = get_hostpolicy_context(/*require_runtime*/ true);
+        const std::shared_ptr<hostpolicy_context_t> context = get_hostpolicy_context(/*require_runtime*/ true);
         assert(context != nullptr);
```

Comment:
Even though you didn't change this assertion, is there a possibility it could be hit here?

Reply:
Ah, yeah. Changed to check for null context / return invalid state before calling this.